base/libc.jl

```diff
@@ -493,6 +493,26 @@ struct Group
     mem::Vector{String}
 end
 
+# Gets a subset of the password file entry
+# (e.g., uid and guid are set to -1 on Windows)
+function getpw()
+    ref_pd = Ref(Cpasswd())
+    ret = ccall(:uv_os_get_passwd, Cint, (Ref{Cpasswd},), ref_pd)
+    ret != 0 && Base.uv_error("getpw", ret)
```

Comment:
```suggestion
    Base.uv_error("getpw", ret)
```
The `ret != 0` check is unnecessary because `uv_error` only throws if `ret < 0`.   (Similarly in `getpwuid`.)

Reply:
I think it is necessary in `getpwuid` because we just want to return `nothing` if `throw_error=false`.